app/models/claims/claim.rb

```diff
@@ -28,8 +28,8 @@
 #  fk_rails_...  (provider_id => providers.id)
 #  fk_rails_...  (school_id => schools.id)
 #
-class Claim < ApplicationRecord
-  belongs_to :school, class_name: "Claims::School"
```

Comment:
I'm wondering why do we need a claim to belong to a school and not a claim school?

Reply:
This happens by default. Since the `Claims::Claim` model is a part of the `Claims` namespace, its associations lookup the same namespace first. `Claims::School` -> `School` -> "Not Found".

Reply:
Ah OK 👍🏻